data retrieval/src/main/java/de/tinf15b4/kino/retrieval/scraper/AbstractCinemaScraper.java

```diff
@@ -40,39 +40,46 @@
 
     public AbstractCinemaScraper() {
         tmdb = new TmdbDataRetriever();
+        cinema = new Cinema();
+        cinema.setName("UNKNOWN");
```

Comment:
Do we really want this? I believe we should rather crash and burn if the subclass doesn't set a cinema